src/ResourceManager/Sql/Commands.Sql/Properties/Resources.Designer.cs

```diff
@@ -673,6 +673,15 @@ internal class Resources {
         }
         
         /// <summary>
+        ///   Looks up a localized string similar to Warning: Cutting off access to the key &apos;{0}&apos; may result in data loss on this server. Learn about best practices at https://docs.microsoft.com/sql/relational-databases/security/encryption/transparent-data-encryption-byok-azure-sql. Do you want to proceed?.
```

Comment:
This is a great message but it should be aka.ms link because the page will inevitably moved